src/app/services/actions/profile-action-creators.ts

```diff
@@ -43,6 +50,12 @@ export function getProfileInfo(query: IQuery): Function {
         const contentType = respHeaders['content-type'];
         const isImageResult = isImageResponse(contentType);
 
+        try {
+          dispatch(profileTypeSuccess(json.account[0].source.type[0]))
+        } catch (error) {
+          console.log(error);
+        }
+
```

Comment:
Since the regular v1.0 graph URL, `${GRAPH_URL}/v1.0/me`, doesn't include the field for account type (MSA or AAD), it crashes and freezes the Graph Explorer. Maybe it's better that I create a function to check if its the beta URL, rather than doing try catch?

Reply:
Fixed in these commits, https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/1008/commits/d492b75ff8a10ba86b4dba7e86ff032a355cd2b0, https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/1008/commits/f0a0105d6c277638f9d4e58da8ebffaad89aaf24